server/documents/views/card.html.eco

```diff
@@ -264,7 +264,8 @@ themes      : ['Default', 'Basic', 'Classic', 'Instagram']
   <div class="another example">
     <div class="evaluated code" data-type="javascript">
     $('.special.cards .image').dimmer({
-      on: 'hover'
+    // As hover is not working on mobile, you might use click on those devices as fallback
+      on: 'ontouchstart' in document.documentElement ? 'click' : 'hover'
```

Comment:
nit) This is fine for many cases, but more trick required for touchable devices with mouse support (like Microsoft Surface) probably

Reply:
Yes, once we figured out a reliable solution (requested for other components) we can adjust this hint.